style.css

```diff
@@ -443,3 +475,465 @@ textarea::placeholder {
+  margin-top: 1.5rem;
+  margin-bottom: 1.5rem;
+  border-radius: 0.5rem;
+}
+
+.title-desktop-left {
+  grid-area: 1 / 1 / 2 / 2;
+  margin-top: 3rem;
+  margin-left: 3rem;
+}
+
+.frame-desktop-left {
+  grid-area: 2 / 1 / 3 / 2;
+  margin-top: 0;
+  margin-left: 3rem;
+}
+
+.primary-project-text-desktop-left {
+  grid-area: 6 / 1 / 7 / 2;
+  margin-left: 3rem;
+}
+
+.tags-desktop-left {
+  grid-area: 8 / 1 / 9 / 2;
+  margin-left: 3rem;
+}
+
+#tags-desktop-uber {
+  grid-area: 7 / 1 / 8 / 2;
+  margin-left: 3rem;
+}
+
+.button-desktop-left {
+  grid-area: 11 / 1 / 12 / 2;
+  margin-left: 3rem;
+}
+
+#button-desktop-uber {
+  grid-area: 9 / 1 / 10 / 2;
+  margin-left: 3rem;
+}
+
+.project-title-desktop {
+  font-family: 'Poppins', sans-serif;
+  font-size: 2.5rem;
+  font-weight: 700;
+  text-align: left;
+  color: #091e42;
+  margin-top: 3rem;
+  padding-bottom: 0;
+}
+
+#uber-parag {
+  margin-bottom: 0;
+}
+
+.primary-project-text-desktop {
+  font-family: 'Poppins', sans-serif;
+  font-size: 1rem;
+  font-weight: 400;
+  line-height: 1.5rem;
+  text-align: left;
+  height: 3.125rem;
+  width: 31rem;
+  margin-bottom: 2rem;
+}
+
+.frame-2-desktop {
+  font-size: 1.125rem;
+  font-weight: 400;
+  line-height: 1.5rem;
+  display: flex;
+  justify-content: flex-start;
+  list-style-type: none;
+  font-family: 'Poppins', sans-serif;
+  color: #6b778c;
+}
+
+.frame-2-desktop li {
+  margin-right: 0.75rem;
+}
+
+.frame-first-attribute {
+  color: #344563;
+  font-size: 1.125rem;
+  font-weight: 500;
+  line-height: 1.5rem;
+}
+
+.tags-desktop {
+  padding: 0.25rem 0.75rem;
+  background-color: #ebebff;
+  border-radius: 0.5rem;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 0.75rem;
+  margin-right: 0.5rem;
+  margin-left: 0.5rem;
+  color: #6070ff;
+  height: 1.5rem;
+}
+
+.button-grid-desktop {
+  margin: 0;
+  border-radius: 1rem;
+  padding: 0.75rem;
+  border: 0.0625rem solid #396df2;
+  color: #396df2;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 1.0625rem;
+  background-color: #fff;
+}
+
+.button-grid-desktop:hover {
+  background-color: #6070ff;
+  color: #fff;
+}
+
+.button-grid-desktop:disabled {
+  border: 1px solid #5e6c84;
+  color: #5e6c84;
+}
+
+.button-grid-desktop:active {
+  color: #fff;
+  background-color: #2230d2;
+}
+
+.button-facebook {
+  color: #fff;
+  background-color: #2230d2;
+  font-size: 1.0625rem;
+}
+
+.button-facebook:active {
+  background-color: #fff;
+  color: #2230d2;
+}
+
+.button-facebook:hover {
+  background-color: #6070ff;
+  color: #fff;
+}
+
+.button-facebook:disabled {
+  border: 1px solid #5e6c84;
+  color: #5e6c84;
+}
+
+#hand-cursor {
+  width: 3.5625rem;
+  height: auto;
+  margin-left: 275px;
+  position: relative;
+  bottom: -1.5rem;
+}
+
+@media screen and (max-width: 768px) {
+  .mobile-hide {
+    display: none;
+  }
+}
+
+@media screen and (min-width: 768px) {
+  body {
+    width: 1440px;
```

Comment:
- [ ] I kindly suggest that you make use of percentages for your width because it'll prevent the current horizontal scroll by keeping a responsive width for all devices.

## FOR EXAMPLE
- Instead of applying 1444px of width to your body selector, you can make use of 100% if the body need 100% of the width on the page.
- Using a width of 1444px would make make your page have an horizontal scroll on desktop devices lower than 1444px

https://www.loom.com/share/a308e6b633684652b15a858c7f8b6ec1

https://www.loom.com/share/707969ee25cc43f18ec74443b5466216